sdk/src/instruction.rs

```diff
@@ -116,7 +116,7 @@ pub enum InstructionError {
     /// by the Solana runtime. A program-specific error may be any type that is represented as
     /// or serialized to a u32 integer.
     #[error("program error: {0:#x}")]
-    CustomError(u32),
+    Custom(u32),
```

Comment:
the error message above says "program error", should we rename `Custom` to `Program`?

Reply:
Never mind, I think `Custom` makes more sense. Could you update the error message to "custom program error"?